src/db.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize({
```

Comment:
The Sequelize constructor is missing the 'database' parameter as the first argument. It should be passed explicitly before the configuration object.